watertap/unit_models/nanofiltration_DSPMDE_0D.py

```diff
@@ -1498,12 +1495,11 @@ def initialize_build(
                 self._automate_rescale_variables()
         # ---------------------------------------------------------------------
         # Solve unit attempt 1
-        # self.eq_solute_solvent_flux.deactivate()
```

Comment:
As an aside, it's interesting that this approach doesn't seem to work for this model. We use something similar in OARO, which was inspired by the IDAES heat exchanger initialization.

Reply:
It may be helpful for the model; I don't remember rechecking after the fact. This was something that I had in place when I still debugging the model at a more fundamental level. Hard to recall, but once I found the real issues with the model, I think I found that this was no longer needed, commented out, but never deleted. We could try it out again to see if it helps with stability, but would prefer to reserve for another PR.